data/data/azure/variables-azure.tf

```diff
@@ -176,3 +176,15 @@ variable "azure_control_plane_vm_networking_type" {
   type        = bool
   description = "Whether to enable accelerated networking on control plane nodes."
 }
+
+variable "azure_vm_architecture" {
+  type        = string
+  description = "Architecture of the VMs - used when creating images in the image gallery"
+}
+
+variable "azure_image_release" {
+  type        = string
+  description = "Architecture of the VMs - used when creating images in the image gallery"
```

Comment:
Seems a c&p error from the above definition?

Reply:
sigh..yeah